packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx

```diff
@@ -24,34 +24,38 @@ export const DocumentLayout = () => {
   const space = spaces.find((space) => abbreviateKey(space.key) === spaceKey && space.state.get() === SpaceState.READY);
 
   const [searchParams] = useSearchParams();
+  const navigate = useNavigate();
   const spaceInvitationCode = searchParams.get('spaceInvitationCode');
   const haloInvitationCode = searchParams.get('haloInvitationCode');
 
   return (
-    <ShellProvider
-      space={space}
-      spaceInvitationCode={spaceInvitationCode}
-      haloInvitationCode={haloInvitationCode}
-      onJoinedSpace={(spaceKey) => {
-        console.log('[joined space]', spaceKey);
-      }}
-    >
-      <OctokitProvider>
-        <PanelSidebarProvider
-          slots={{
-            content: {
-              children: <SidebarContent />,
-              className: mx(defaultOsButtonColors, shadow, 'backdrop-blur overflow-visible'),
-              onOpenAutoFocus: (event) => event.preventDefault(),
-              onCloseAutoFocus: (event) => event.preventDefault()
-            },
-            main: { role: 'main', className: 'min-bs-full' }
-          }}
-        >
-          <Outlet context={{ space }} />
-          <SidebarToggle />
-        </PanelSidebarProvider>
-      </OctokitProvider>
-    </ShellProvider>
+    <>
+      <span className='sr-only' data-testid='composer.pageReady' />
```

Comment:
in kai I found this morning that I could look for `dxos-shell` which is the testid we add to the shell iframe